Apps/Playground/scripts/postinstall.js

```diff
@@ -0,0 +1,10 @@
+const shelljs = require('shelljs');
+const output = shelljs.exec('npm list -g npm');
+const parts = output.split("npm@");
+const version = parts[1].trim();
+
+// Check if using npm version 7+
+if (version.charAt(0) == '7')

```

Comment:
You could also do something like `!!version.match(/6\.13\..*/g)`, or import semver

Reply:
this should now be updated